Documentation/WritingDocForExtension/Migrate.rst

```diff
@@ -32,32 +32,226 @@ a migration tool. This tool can be used to automatically create a
+:file:`Settings.cfg` is currently placed in.
 
-    docker run --rm --pull always \
-        -v $(pwd):/project \
-        -it ghcr.io/typo3-documentation/render-guides:latest \
-        migrate Documentation
+After the migration is performed, you will see output about which settings were
+converted, if some old settings were discarded, or errors occurred.
 
 Try out the rendering locally
 =============================
 
 Use our Docker container to render your documentation locally. Read more about
 :ref:`local rendering <rendering-docs>`.
 
-Improve your documentation to render without warning
-====================================================
+..  _migrate-detailed-steps:
+
+Further steps to adapt to the new rendering
+===========================================
+
+You should perform the following tasks to conclude the migration to the
+new rendering tool:
+
+..  rst-class:: bignums
+
+1.  Improve your documentation to render without warning
+
+    Rendering your documentation should not yield any warnings or errors.
+
+    If you get error messages, often they refer to wrong indentation, missing
+    :ref:`interlinks <settings-guides-interlink-mapping>`, orphaned files or
+    outdated ReST identifiers.
+
+    If you are unable to address a warning/error with changes in your documentation
+    feel free to ask in Slack channel #typo3-documentation (see :ref:`how-to-get-help`).
+
+    If you believe you found a specific bug in the new PHP-based rendering, please open
+    an `Issue on GitHub <https://github.com/TYPO3-Documentation/render-guides/issues>`__.
+
+2.  Remove outdated files
+
+    After you have created the :file:`guides.xml` file, you can remove the old
+    :file:`Settings.cfg` file.
+
+    You can also delete the files :file:`genindex.rst` which was previously used
+    to generate an index.
+
+3.  Adapt :file:`Includes.rst.txt`
+
+    The main documentation directory can contain a file :file:`Includes.rst.txt`
+    to include any fixed text, which will be placed on every page of your rendered
+    documentation.
+
+    Previously it was also used to define a list of utilized directives/roles.
+
+    You can either remove that file, or add your fixed text to it. If you remove
+    the file, remember to also remove all references pointing to that file, like:
+
+    ..  code-block:: text
+        :caption: Documentation/Index.rst
+
+        ..  include:: /Includes.rst.txt
+
+    Most official documentation uses this as the stub of the file:
+
+    ..  code-block:: text
+        :caption: Documentation/Includes.rst.txt
+
+        ..  You can put central messages to display on all pages here
+
+4.  Remove the entry `genindex` from :file:`Index.rst` (Index/Glossary)
+
+    If you previously had a :file:`genindex.rst` file, this optional index
+    (or glossary) was rendered as a page through an entry in the file :file:`Index.rst`
+    like this:
+
+    ..  code-block:: text
```

Comment:
I would like to see here a diff instead of a text. This makes clearer which file to remove at which position.

Reply:
How would you format a diff? I just added a "-" now in front of it and changed "code-block::" to "diff". Doesn't look that much different though. The line emphasize should highlight the removal already though:

<img width="768" alt="Screenshot 2024-03-13 at 09 21 46" src="https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-HowToDocument/assets/273326/1556e5b3-2e7b-4fe7-9324-15c89ac5e56c">

Reply:
Remove the emphasize-lines. Add the `-` in the first column and indent all other lines by one column to the right

Reply:
Ack, done!